om-2.0.rdf

```diff
@@ -42704,6 +42789,7 @@
     <om:usesUnit rdf:resource="&om;ampere"/>
     <om:usesUnit rdf:resource="&om;milliampere"/>
     <om:usesUnit rdf:resource="&om;volt"/>
+    <om:usesUnit rdf:resource="&om;voltampere"/>
     <om:usesUnit rdf:resource="&om;millivolt"/>

```

Comment:
For consistency, you should probably add kilovoltampere.

Reply:
I have added Kilo-voltampere (kVA), Megavolt-ampere (mVA), Kilo-voltampere reactive (kVAR), and Megavolt-ampere reactive (mVAR) ontologies for consistency.